### PR TITLE
Internal: Disabled flaky test workflow

### PR DESCRIPTION
**Pull request by dmaziar**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: .github/workflows/playwright.yml [hunk 1]
```diff
@@ -23,6 +23,11 @@ on:
             - test-results/
             - tests/playwright/blob-report
             - allure-results
+        fail_fast:
+            type: boolean
+            required: true
+            description: 'Run all tests'
```
Comment (yotamselementor) [inline .github/workflows/playwright.yml:29]:
```suggestion
            description: 'Cancel tests when one of them fails'
```

File: .github/workflows/playwright.yml [hunk 2]
```diff
@@ -150,22 +156,16 @@ jobs:
...
           TEST_SUITE: "@${{matrix.testSuite}}"
-        run: npm run test:playwright -- --reporter=${{inputs.reporter || 'list'}}
+        run: npm run test:playwright -- --reporter=${{inputs.reporter || 'list,github'}}
       - name: Run element regression tests
         if: ${{matrix.testSuite =='elements-regression'}}
-        run: npm run test:playwright:elements-regression -- --reporter=${{inputs.reporter || 'list'}}
+        run: npm run test:playwright:elements-regression -- --reporter=${{inputs.reporter || 'list,github'}}
       - uses: actions/upload-artifact@v3
-        if: always()
+        if: ${{ inputs.fail_fast == false || failure() }}
```
Comment (yotamselementor) [inline .github/workflows/playwright.yml:164]: Why not upload the results anyhow? If we don't have these its hard to debug

**Review by yotamselementor — CHANGES_REQUESTED** [review]

File: .github/workflows/playwright.yml (hunk 1 above)
Comment (dmaziar) [reply .github/workflows/playwright.yml]: fixed

File: .github/workflows/playwright.yml (hunk 2 above)
Comment (dmaziar) [reply .github/workflows/playwright.yml]:
```
(https://github.com/elementor/elementor/actions/runs/7685951345/job/20944299627#step:14:8)
No files were found with the provided path: test-results/. No artifacts will be uploaded.
```
Fixed. But we will have this warnings

**Review by yotamselementor — APPROVED** [review]

**Pull request merged**